example-deployment/docker/dashboard.dockerfile

```diff
@@ -20,8 +21,8 @@ COPY --from=0 /root/rmf-web/packages/dashboard/build /usr/share/nginx/html/dashb
 SHELL ["bash", "-c"]
 RUN echo -e 'server {\n\
   location / {\n\
-    root /usr/share/nginx/html;\n\
-    index index.html index.htm;\n\
-    try_files $uri /dashboard/index.html;\n\
+  root /usr/share/nginx/html;\n\
+  index index.html index.htm;\n\
+  try_files $uri /dashboard/index.html;\n\
```

Comment:
Was this changed by some auto formatter? The indent is wrong.

Reply:
Yeah, it was prettier. I checkout this file from the main branch